fedocal/__init__.py

```diff
@@ -1030,7 +1038,9 @@ def view_meeting_page(meeting_id, full):
         tzone=tzone,
         title=meeting.meeting_name,
         editor=editor,
-        from_date=from_date)
+        from_date=from_date,
+        reminder_at_options=APP.config.get('ICAL_REMINDER_AT_OPTIONS') or []
```

Comment:
We shouldn't be able to have
``
config[ICAL_REMINDER_AT_OPTIONS'] = None
``
as it would mean someone edited the default_config.py instead of overriding it.

So +1 on not being tolerant to this, but in the worst case, I think I would prefer a ``if reminder_at_options`` lower down the stack.